cglib/src/main/java/net/sf/cglib/core/ClassLoaderAwareClassWriter.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2003,2004 The Apache Software Foundation
```

Comment:
Should this be 2018?

Reply:
That's an interesting question. This particular code wasn't actually developed by Apache Foundation, so it's technically incorrect to include its license header here - so it seems, that it's better just to delete it.